group_boards.go

```diff
@@ -0,0 +1,248 @@
+//
+// Copyright 2018, Patrick Webster
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// GroupIssueBoardsService handles communication with the group issue board related
+// methods of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html
+type GroupIssueBoardsService struct {
+	client *Client
+}
+
+// GroupIssueBoard represents a GitLab group issue board.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/group_boards.html
+type GroupIssueBoard struct {
+	ID        int          `json:"id"`
+	Name      string       `json:"name"`
+	Group     *Group       `json:"group"`
+	Milestone *Milestone   `json:"milestone"`
+	Lists     []*BoardList `json:"lists"`
+}
```

Comment:
In the examples in the docs I don't see the `name` and `milestone` fields. Are you sure they are returned by the API?

Reply:
Interesting -- they're not in the ce api, but they definitely are in the ee api which we use in house -- https://docs.gitlab.com/ee/api/group_boards.html#group-board

Reply:
Check! Didn't check those, but then it's fine 👍